cfme/tests/control/test_basic.py

```diff
@@ -616,7 +610,7 @@ def test_alert_crud(appliance):
 
 
 @pytest.mark.tier(3)
-@pytest.mark.meta(blockers=[1303645], automates=[1303645])
+@pytest.mark.meta(automates=[1303645])
```

Comment:
wow :smiley: do we use it elsewhere?

Reply:
A bit - I want to script moving everything from docblocks into this marker